### DISCUSSION TITLE
video-like controls on GIFs

### DISCUSSION THREAD

**Discussion by iTrooz** [Docile-Alligator/Infinity-For-Reddit#910] [Ideas]
Currently, gifs just autoplay, and you can't pause/rewind/etc.. them, and I think controls like videos would be a good addition

I'm willing to work on this and submit a PR if you give me the get-go

**Comment by Wladefant** [comment]
@iTrooz just found a real gif after a long search: https://www.reddit.com/r/gifs/comments/vwyx4h/hubble_image_of_deep_space_vs_james_webbs_image/

I agree that gifs need all the video player features that videos without audio have

**Comment by iTrooz** [comment]
Indeed !
I'm just waiting for the approval of a maintainer to start working on it, wouldn't want the merge to be refused ahah

**Comment by Wladefant** [comment]
> Indeed !
> I'm just waiting for the approval of a maintainer to start working on it, wouldn't want the merge to be refused ahah
> 
> 

Just wrote him about you, it should be approved soon

**Comment by Docile-Alligator** [comment]
@iTrooz Thank you! Currently, Infinity does not distinguish between gifs and images. So, maybe you need to find a way to know if the media is a gif first.

**Comment by Wladefant** [comment]
> @iTrooz Thank you! Currently, Infinity does not distinguish between gifs and images. So, maybe you need to find a way to know if the media is a gif first.

I haven't looked in the code but shouldn't it already distinguish, because videos have the labeling video and gifs have the labeling gif

**Comment by Wladefant** [comment]
Implementation: #1126
